R/volcanoPlot.R

```diff
@@ -46,36 +46,30 @@
 #'    not in topTable output by default so the user has to bind this column
 #'    to the dataframe in advance.  This column will be used to label
 #'    significantly changed points.
-#' @param pthresholdLine Color for a horizontal line at the p-threshold (Default
-#'   = NULL (disabled))
-#' @param symbolSize Size of symbols for Up, no change, and Down. default = c(4,
-#'        3.99, 4); Note: All three cannot be the same size. Decimal values are acceptable to help offset that
+#' @param symbolSize Size of symbols for Up, no change, and Down. default = c(10, 4, 10);
+#'  Note: All three cannot be the same size. Decimal values are acceptable to help offset that
 #'        (e.g. 4, 4.1, 4.2).
 #' @param symbolShape Shape of the symbols for Up, no change, and Down; Default =
-#'        c(21, 1, 21) (1 = open circle, 21 = fillable open circle); Note: The same symbol shape cannot
+#'        c("circle", "circle", "circle"); Note: The same symbol shape cannot
 #'        be selected for all three symbols. See
 #'        \url{http://www.cookbook-r.com/Graphs/Shapes_and_line_types}
-#' @param symbolColor c(Up, NoChange, Down); default = c("black", "grey25",
-#'   "grey0") See \url{http://research.stowers-institute.org/efg/R/Color/Chart}
+#' @param symbolColor c(Up, NoChange, Down); default = c("red3", "grey25", "deepskyblue4")
+#'  See \url{http://research.stowers-institute.org/efg/R/Color/Chart}
 #'   Note: Colors cannot be duplicated.
-#' @param symbolFill Set the fill color for the symbols. Note only symbols 21-25
+#' @param symbolColor Set the fill color for the symbols. Note only symbols 21-25
 #'   are fillable. This will have no effect on other symbols. Default =
 #'   c("red3", "grey25", "deepskyblue4") Note: Colors cannot be duplicated.
-#' @param alpha Controls the transparency of the plotted points (range: 0-1;
-#'   default = 0.7)
+#' @param transparency Controls the transparency of the plotted points (range: 0-1;
+#'   default = 0.5)
 #' @param sizeByIntensity If TRUE, creates a column to support sizeByIntensity. (Default = TRUE)
 #' @param foldChangeLines Position of reference vertical lines for fold change
 #'   (Default = log2(1.5); NULL disables)
 #' @param legendPosition One of "top", "bottom", "left", "right", "ne", "se",
```

Comment:
This is not updated. I am seeing a different set of values in the validation for legendPosition.